### ISSUE TITLE
Can't find HTML files if the outDir property has a relative path

### ISSUE THREAD

**Issue by felixzapata** [zhuweiyou/vite-plugin-minify#1]
Hi, with this configuration for the build:

```
  root: './src',
  base: '/',
.....
build: {
    outDir: '../dist',
    assetsDir: 'assets',
    emptyOutDir: true
  },
```
the plugin can't find any HTML files to read. 

At the end the plugin tries to read `glob('../dist/**/*.html', (err, files) => {` but the result is zero files.

**Comment by zhuweiyou** [comment]
You should use `resolve`

```js
import { resolve } from 'path'

const r = p => resolve(__dirname, p)

root: r('./src')
...
build: {
  outDir: r('../dist')
}
```

**Comment by felixzapata** [comment]
I've just created a pull request with another solution.

**Comment by zhuweiyou** [comment]
Fix it in the #3
